src/models/impact/impactTarget.ts

```diff
@@ -0,0 +1,27 @@
+import { IMPACT_ACTIONS } from "../../components/Impact/constants";
+
+export interface IImpactTarget {
+	id?: number;
+	name: string;
+	description?: string;
+	target_value: number | string;
+	impactCategory?: number | string;
+	impactUnit?: number | string;
+	impact_category_unit: number | string;
+	project?: number | string;
+}
+
+export type ImpactTargetProps =
```

Comment:
some of the keys are common in obth cases. Can we initialize the monly once?

Reply:
done